mkdocs.yml

```diff
@@ -61,6 +61,8 @@ nav:
       - ... | flat | cookbook/languages/*
     - Builds:
       - ... | flat | cookbook/builds/*
+    - Infrastructure:
```

Comment:
Maybe we should call this `Publish` instead? The infrastructure is only relevant for publishing (that's my thinking).

Reply:
I'm fine with that and don't feel strongly at all about the structure. I was thinking infra vs languages, but having build and publish there is perfectly sensible as well.